tests/test_utils.py

```diff
@@ -347,3 +349,18 @@ def test_all_literal_values():
 
     L312 = Literal['3', Literal[L1, L2]]
     assert sorted(all_literal_values(L312)) == sorted(('1', '2', '3'))
+
+
+def test_get_caller_module_name():
+    assert get_caller_module_name() == '_pytest.python'
+
+    def get_current_module_name():
+        return get_caller_module_name()
+
+    assert get_current_module_name() == __name__
+
+
+def test_get_caller_module_name_error():
+    with mock.patch('inspect.getmodule', return_value=None):
+        with pytest.raises(LookupError, match='Could not find caller module'):
+            get_caller_module_name()
```

Comment:
```suggestion
def test_get_caller_module_name_error(mocker):
    mocker.patch('inspect.getmodule', return_value=None)
    with pytest.raises(LookupError, match='Could not find caller module'):
        get_caller_module_name()
```
Imo it's better to use directly `mocker` fixture and not import `unittest.mock`